PrintTheReverseOfThreeDigitNumber.java

```diff
@@ -0,0 +1,23 @@
+import java.util.Scanner;
+public class PrintTheReverseOfThreeDigitNumber {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Input 3 digits number");
+        int ThreeDigitsNumber = sc.nextInt();
+        if (ThreeDigitsNumber > 999 || ThreeDigitsNumber < 100) {
+            System.exit(1);
+        } else {
+            int ThirdDigit = ((ThreeDigitsNumber % 100) % 10);
+            int SecondDigit = ((ThreeDigitsNumber % 100) / 10);
+            int FirstDigit = (ThreeDigitsNumber / 100);
+            int ReversedNumber = (ThirdDigit * 100 + SecondDigit * 10 + FirstDigit);
+            if (ThirdDigit == 0) {
```

Comment:
This is correct, no objections. Just a small hint that when you got the value of `ThirdDigit` variable, you could do the check right away in advance and not calculate `SecondDigit` and `FirstDigit` if it was negative.

Reply:
Understood